NBitcoin/Network.cs

```diff
@@ -2413,16 +2413,20 @@ public BitcoinScriptAddress CreateBitcoinScriptAddress(string base58)
 			return null;
 		}
 
-		public IBitcoinString Parse(string str)
+		public IBitcoinString Parse(string str, Type? targetType = null)
 		{
-			return Parse<IBitcoinString>(str, this);
+			return Parse<IBitcoinString>(str, this, targetType);
 		}
-		public T Parse<T>(string str) where T : IBitcoinString
+		public T Parse<T>(string str, Type? targetType = null) where T : IBitcoinString

```

Comment:
the `targetType` makes no sense here

Reply:
`Parse<T>` should call `Parse` not the reverse.

Reply:
fixeed